scripts/update_review_app.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ev
+
+# Bail out of script on first expression failure and echo the commands as
+# they are being run.
+set -ev
+
+NAME="$1"
+
+if test -z $NAME; then
```

Comment:
We should wrap any variable in double quotes to prevent globbing and word splitting.

Reply:
Check out L9 above – we're wrapping `$1` in double quotes, so I think that's sufficient?